.github/workflows/pr-build-and-e2e-tests.yml

```diff
@@ -71,6 +71,16 @@ jobs:
               id: run_playwright_e2e_tests
               run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js --shard ${{ matrix.shard.name }}
 
+            - name: Run Playwright tests on chrome-new-editor.
+              timeout-minutes: 60
+              id: run_playwright_e2e_tests_new_editor
+              env:
+                  USE_WP_ENV: 1
+                  E2E_MAX_FAILURES: 15
+                  FORCE_COLOR: 1
+              working-directory: plugins/woocommerce
+              run: pnpm exec playwright test --project=chrome-new-editor --config=tests/e2e-pw/playwright.config.js --shard ${{ matrix.shard.name }}
+
```

Comment:
Instead of adding this here I think we should move this down to it's own task, that way its also more clearly visible within the `actions` thread and runs in parallel with the other actions ( on separate instances ).
So we did add something like this:
```
e2e-tests-run:
        name: Runs E2E new product editor tests.
        runs-on: ubuntu-latest
        timeout-minutes: 60
        strategy:
            fail-fast: false
        permissions:
            contents: read
        env:
            ALLURE_RESULTS_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/test-results/allure-results
            ALLURE_REPORT_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/test-results/allure-report
        outputs:
            E2E_GRAND_TOTAL: ${{ steps.count_e2e_total.outputs.E2E_GRAND_TOTAL }}
        steps:
            - uses: actions/checkout@v3

            - name: Setup WooCommerce Monorepo
              uses: ./.github/actions/setup-woocommerce-monorepo
              with:
                  install: '@woocommerce/plugin-woocommerce...'
                  build: '@woocommerce/plugin-woocommerce'

            - name: Load docker images and start containers.
              working-directory: plugins/woocommerce
              env:
                  WP_ENV_PHP_VERSION: 7.4
              run: pnpm env:test

            - name: Download and install Chromium browser.
              working-directory: plugins/woocommerce
              run: pnpm exec playwright install chromium

            - name: Get total number of Playwright E2E tests to be run.
              id: count_e2e_total
              working-directory: plugins/woocommerce
              run: |
                  TOTAL_STR=$(pnpm exec playwright test --project=chrome-new-editor --config=tests/e2e-pw/playwright.config.js --list | grep "Total:")
                  NO_PREFIX=${TOTAL_STR#*"Total: "}
                  COUNT=${NO_PREFIX%" tests in"*}
                  echo "E2E_GRAND_TOTAL=$COUNT" >> $GITHUB_OUTPUT

            - name: Run Playwright tests
              working-directory: plugins/woocommerce
              env:
                  USE_WP_ENV: 1
                  E2E_MAX_FAILURES: 15
                  FORCE_COLOR: 1
              id: run_playwright_e2e_tests
              run: pnpm exec playwright test --project=chrome-new-editor --config=tests/e2e-pw/playwright.config.js

            - name: Upload reports to GitHub Actions Artifacts
              if: always()
              uses: actions/upload-artifact@v4
              with:
                  name: all-blob-reports-product-editor
                  path: ${{ env.ALLURE_RESULTS_DIR }}
                  retention-days: 1
                  compression-level: 9
```

Reply:
Good call! This approach works great! I added it in the commit https://github.com/woocommerce/woocommerce/pull/41601/commits/daf60655ba8a533b0ec8e8c9c6e76fffc476dd06